src/main/java/Models/Prescription/Prescription.java

```diff
@@ -0,0 +1,49 @@
+package Models.Prescription;
+
+/**
+ *
+ * @author Bredan
+ */
+
+public class Prescription {
+    private final Integer patientID;
```

Comment:
This can be a User type called `patient` instead of a patientID

Reply:
Resolved!